.github/dependabot.yml

```diff
@@ -229,73 +229,119 @@ updates:
       - "no-changelog"
 
   - package-ecosystem: npm
-    directory: "/web/packages/build"
+    directory: "/"
```

Comment:
It's just the diffing getting confused because I added a root directory as well (after reading through issues, we needed to do this to keep the yarn lock updated)

The /build repo is still tracked underneath this